src/library_syscall.js

```diff
@@ -52,7 +52,7 @@ var SyscallsLibrary = {
         throw e;
       }
       {{{ makeSetValue('buf', C_STRUCTS.stat.st_dev, 'stat.dev', 'i32') }}};
-      {{{ makeSetValue('buf', C_STRUCTS.stat.__st_ino_truncated, 'stat.ino', 'i32') }}};
+      {{{ makeSetValue('buf', C_STRUCTS.stat.__st_ino_truncated, 'stat.ino >>> 0', 'i32') }}};
```

Comment:
What effect does this have? `HEAP32[x] = y` will automatically do the effect of `|0` to `y` as it is written, won't it? Is there some subtle signed/unsigned issue I am missing?

Reply:
Oh, is it hitting the recent assertion you added @sbc100 that checks the value fits?

Reply:
Yup.. nice to see them doing their job!